buildSrc/src/main/java/com/uber/okbuck/core/model/android/AndroidTarget.java

```diff
@@ -412,9 +412,12 @@ void ensureManifest() {
   }
 
   @Override
-  protected List<String> getKotlinCompilerOptions() {
+  protected List<String> getKotlinCompilerOptions(boolean testOnly) {
+    if (!isKotlin) {
```

Comment:
should this is `isKotlinAndroid`?

Reply:
No. Kotlin compiler options can apply to both regular and android projects. I differentiated and added a `isKotlin` field to the base Target class and renamed the android target's to `isKotlinAndroid`

Reply:
I did remove it though because it's redundant here